tictactoe/adapter-implementation/src/main/kotlin/com/anaplan/engineering/azuki/tictactoe/adapter/implementation/action/PlayerMoveAction.kt

```diff
@@ -10,7 +10,7 @@ class PlayerMoveAction(gameName: String, playerName: String, position: Position)
 
     override fun act(env: ExecutionEnvironment) {
         env.modifyGame(gameName) {
-            move(toPlayer(playerName), position.col, position.row)
+            move(toPlayer(playerName), position.col - 1, position.row - 1)
```

Comment:
I didn't know whether you intended to have `Game` intentionally zero-based so that the adapter had to do some adapting, but I've assumed that was the case :)